config/a10_a100_config.json

```diff
@@ -1,6 +1,6 @@
 {
     "bf16": {
-      "enabled": "auto"
+      "enabled": true
```

Comment:
I'm not sure if we have to explicitly disable fp16 in this config too. I kind of doubt it. If it worked for you that's fine.

Reply:
I haven't been able to test on an A100 - I haven't been able to get a hold of one yet.

Reply:
Strangely this setting needs to stay as "auto". Explicitly setting this to `true` ends up generating an error. Something about a mismatch between the parameter names between deepspeed and Huggingface.

Reply:
Is it `true` vs `"true"`?